patch/kconfig-inclusions

```diff
@@ -107,6 +115,12 @@ CONFIG_SENSORS_LM25066=m
 CONFIG_SENSORS_UCD9000=m
 CONFIG_SENSORS_UCD9200=m
 CONFIG_THERMAL_OF=y
+CONFIG_SPI_AMD=m
+CONFIG_PINCTRL_AMD=y
+CONFIG_EDAC_AMD64=m
+CONFIG_AMD_XGBE_DCB=y
+CONFIG_AMD_XGBE_HAVE_ECC=y
```

Comment:
This config looks like something that shouldn't be directly configured, but should rather be set by some other config.

Reply:
removed